### PR TITLE
Port remaining jenkins jobs

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

File: jenkins/jobs/lxc-build-android.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    builders:
+      - shell: |-
+          cd /lxc
+          exec sudo /lxc/bin/build-android amd64 https://github.com/lxc/lxc master ${sha1} ${WORKSPACE}/
+
+    publishers:
+      - archive:
+          artifacts: lxc-android.tar.gz
+
+      - workspace-cleanup
```
Comment (stgraber) [inline jenkins/jobs/lxc-build-android.yaml]: Our standard pattern for this is to do:
```
- workspace-cleanup:
  fail-build: false
```

Can you update them all?

File: jenkins/jobs/lxc-build-android.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    parameters:
+      - string:
+          name: sha1
+          description: Hash of the commit to build
+
+      - string:
+          name: branch
+          description: Source git branch
+
+    builders:
```
Comment (stgraber) [inline jenkins/jobs/lxc-build-android.yaml]: Small nit but in the existing jobs, this goes to the top, right after project-type and before properties.

**Comment by stgraber** [comment]
Can you also make the yaml look more similar to our existing jobs so we can diff them without hitting issues due to differing indent?

And it looks like we have all our current jobs running ansicolor so would be nice for the newly ported ones to also use that.

Thanks!

File: jenkins/jobs/lxc-build-android.yaml (hunk 1 above)
Comment (monstermunchkin) [reply jenkins/jobs/lxc-build-android.yaml]: Done

File: jenkins/jobs/lxc-build-android.yaml (hunk 2 above)
Comment (monstermunchkin) [reply jenkins/jobs/lxc-build-android.yaml]: Done

**Comment by monstermunchkin** [comment]
All done

**Pull request merged**